src/analyzer.rs

```diff
@@ -235,7 +244,8 @@ mod test {
 
     #[test]
     fn test_simple_latin() {
-        let analyzer = Analyzer::new(AnalyzerConfig::default());
+        let stop_words = Set::from_iter([""].iter()).unwrap();
```

Comment:
There is a default impl, it's usually better than doing this by hand, here you create one empty key, do you really want that?

Reply:
It you wanted an empty iterator a `None as Option<&str>` would have been better.

Reply:
thanks, I was not sure how to deal with that :)